pkg/daemon/rpm-ostree.go

```diff
@@ -237,92 +175,21 @@ func (r *RpmOstreeClient) IsNewEnoughForLayering() (bool, error) {
 }
 
 // RebaseLayered rebases system or errors if already rebased.
-func (r *RpmOstreeClient) RebaseLayered(imgURL string) (err error) {
+func (r *RpmOstreeClient) RebaseLayered(imgURL string) error {
 	// Try to re-link the merged pull secrets if they exist, since it could have been populated without a daemon reboot
-	useMergedPullSecrets()
+	if err := useMergedPullSecrets(rpmOstreeSystem); err != nil {
+		klog.Errorf("error while linking rpm-ostree pull secrets %v", err)
+	}
 	klog.Infof("Executing rebase to %s", imgURL)
 	return runRpmOstree("rebase", "--experimental", "ostree-unverified-registry:"+imgURL)
 }
 
 // RebaseLayeredFromContainerStorage rebases the system from an existing local container storage image.
-func (r *RpmOstreeClient) RebaseLayeredFromContainerStorage(imgURL string) (err error) {
+func (r *RpmOstreeClient) RebaseLayeredFromContainerStorage(imgURL string) error {
 	// Try to re-link the merged pull secrets if they exist, since it could have been populated without a daemon reboot
-	useMergedPullSecrets()
+	if err := useMergedPullSecrets(rpmOstreeSystem); err != nil {
+		klog.Errorf("error while linking rpm-ostree pull secrets %v", err)
```

Comment:
and same question here as above again, do we just want to log the error and continue?